smart_control/reinforcement_learning/utils/config.py

```diff
@@ -0,0 +1,119 @@
+import os
+from typing import Any
+
+import gin
+import numpy as np
+
+# the following imports are necessary for proper gin setup, even if they aren't referenced
+# do not remove
+from smart_control.reward.electricity_energy_cost import ElectricityEnergyCost
+from smart_control.reward.natural_gas_energy_cost import NaturalGasEnergyCost
+from smart_control.reward.setpoint_energy_carbon_regret import (
+    SetpointEnergyCarbonRegretFunction,
+)
+from smart_control.simulator.air_handler import AirHandler
+from smart_control.simulator.boiler import Boiler
+from smart_control.simulator.building import MaterialProperties
+from smart_control.simulator.hvac_floorplan_based import FloorPlanBasedHvac
+from smart_control.simulator.randomized_arrival_departure_occupancy import (
+    RandomizedArrivalDepartureOccupancy,
+)
+from smart_control.simulator.simulator_building import SimulatorBuilding
+from smart_control.simulator.stochastic_convection_simulator import (
+    StochasticConvectionSimulator,
+)
+from smart_control.simulator.tf_simulator import TFSimulator
+from smart_control.simulator.weather_controller import ReplayWeatherController
+from smart_control.utils import controller_reader, histogram_reducer
+from smart_control.utils.controller_writer import ProtoWriterFactory
+from smart_control.utils.environment_utils import to_timestamp
+from smart_control.utils.observation_normalizer import (
+    StandardScoreObservationNormalizer,
+)
+
+ROOT_DIR = "/home/gabriel-user/projects/sbsim/" # Path to the root directory of the project
```

Comment:
I was originally thinking about using an environment variable here to set the ROOT_DIR, but I will try a relative directory reference instead, which would simplify configuration setup and work automatically.